sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -337,6 +337,32 @@ object RapidsConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(0)
 
+  val OFF_HEAP_LIMIT_ENABLED = conf("spark.rapids.memory.hostOffHeapLimit.enabled")
```

Comment:
Nit: We already have a `spark.rapids.memory.host.` prefix that correlates with `spark.rapids.memory.gpu.`, so recommend leveraging that prefix for these configs as well, e.g.:
```suggestion
  val OFF_HEAP_LIMIT_ENABLED = conf("spark.rapids.memory.host.offHeapLimit.enabled")
```